source/blog/2016-06-14-rspec-3.5-has-been-released.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
The file name should have `3-5`, not `3.5` -- at least if we want to follow the pattern for all the other release posts.